segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -173,11 +178,8 @@ public ChunkedSegmentStorage(int containerId, ChunkStorage chunkStorage, ChunkMe
         this.logPrefix = String.format("ChunkedSegmentStorage[%d]", containerId);
 
         // Now bootstrap
-        log.info("{} STORAGE BOOT: Started.", logPrefix);
-        return this.systemJournal.bootstrap(epoch).thenApplyAsync(v -> {
-            log.info("{} STORAGE BOOT: Ended.", logPrefix);
-            return null;
-        }, executor);
+        log.debug("{} STORAGE BOOT: Started.", logPrefix);
+        return this.systemJournal.bootstrap(epoch).thenRunAsync(() -> log.debug("{} STORAGE BOOT: Ended.", logPrefix), executor);
```

Comment:
Same thing with `log.debug`. DEBUG logging is not a common occurrence, so only add a callback if debug logging is enabled.

Plus, there is absolutely no reason to do `...Async` on these ones. These one-liners will eat up more CPU resources doing context switching than what they actually do need to execute. Just turn this one into `thenRun()` with no executor. It will be executed on the same thread the previous callback finished. 

While this doesn't matter in this particular case, since this is a very infrequent operation, for others it does matter and we don't want to flood the threadpool with useless callbacks like this one.

Reply:
fixed.